minos/networks/__init__.py

```diff
@@ -38,6 +38,7 @@
     CommandResponse,
     CommandResponseException,
     Consumer,
+    Enroute,
```

Comment:
As the `Enroute` variable will be the decorator name, I think it's better to expose the lowecase alias (generated with `enroute = Enroute`)

Reply:
Done !